packages/nushell/run_nushell.bri

```diff
@@ -10,7 +10,7 @@ import nushell from "/";
  * extra dependencies or environment variables using `.dependencies()`
  * or `.env()`, respectively, along with other process options.
  *
- * @description See also `nushellRunnable,` which will return a recipe that
+ * @description See also `nushellRunnable` which will return a recipe that
```

Comment:
Before merging, should we have here `std.nushellRunnable` as in the other TSDoc comment ?


```suggestion
 * @description See also `std.nushellRunnable` which will return a recipe that
```